crypto/src/main/java/tech/pegasys/pantheon/crypto/Blake2bfMessageDigest.java

```diff
@@ -224,7 +224,7 @@ private int bytesToInt(final byte[] bytes) {
     }
 
     private long bytesToLong(final byte[] bytes) {
-      return Pack.bigEndianToLong(bytes, 0);
+      return Pack.littleEndianToLong(bytes, 0);
```

Comment:
Pretty bizarre that the payload is a mix of big endian and little endian values :\

Reply:
Agreed.  But that's what some web3/solidity folks asked for in the EIP and got.